### DISCUSSION TITLE
Manually specify knot locations

### DISCUSSION THREAD

**Discussion by Thom-Teears** [sdmTMB/sdmTMB#204] [Q&A] [answered]
Hello all,
     Just wondering if it is possible to manually place knots at specific locations? I am trying to recreate an analysis that was done in VAST but am having trouble reproducing the same mesh. I have a data frame of knot locations and I can use inla.delaunay to create an identical mesh but I am not sure how to convert that object into a mesh that the sdmTMB fitting function can interpret.

Thanks,
Thom

**Comment by seananderson** [comment, marked as answer]
Hi Thom, you should be able to pass the INLA mesh to the mesh argument in `sdmTMB::make_mesh()`. It will then take the knot locations from that object. There should be an example in the function help file at the end. Does that work for you?

There's also an example of configuring matching VAST and sdmTMB meshes and models in an appendix of the sdmTMB preprint.

**Reply by Thom-Teears** [reply]
Hi Sean,
     Thank you so much for your suggestions. I was able to recreate it following the example in the appendix.

Kind regards,
Thom

**Comment by Thom-Teears** [comment]
Hello,
     I was wondering how I could match up each data point to the corresponding knot locations using the sdmTMB fit object?

Thanks,
Thom
